code/modules/events/heart_attack.dm

```diff
@@ -0,0 +1,29 @@
+/datum/round_event_control/heart_attack
+	name = "Random Heart Attack"
+	typepath = /datum/round_event/heart_attack
+	weight = 20
+	max_occurrences = 2
+	earliest_start = 12000
+	min_players = 40 // To avoid shafting lowpop
+
+/datum/round_event/heart_attack/start()
+	for(var/mob/living/carbon/human/H in shuffle(GLOB.living_mob_list))
```

Comment:
I believe this would infect *the entire crew* with the heart attack disease. For obvious reasons I feel like that's a bit silly.

Reply:
See the break at the bottom?

It's the same as the Appendicitis code.

Reply:
Oh, huh. In that case, it'd only infect *one* person. Seems odd, but given the checks I suppose it's the best way.

Reply:
Yea it would be even odder if people were getting simultaneous heart attacks